docs/pairing.md

```diff
@@ -0,0 +1,114 @@
+# pairing process
+
+The pairing process is a handshake with 4 steps, all done with reported and desired properties:
+1. A device app says "I need a partner" by setting it's reported `serviceRunId` to `null`.
+2. A service app says "I'm available" by setting the devices desired `serviceRunId` to the service app's `runId` value.
+3. The device app says "I choose you" by setting it's reported `serviceRunId` to the `runId` of the chosen service
+4. The service app finalizes the pairing by setting the devices desired `acceptedPairing` value.
```

Comment:
Is there still a pairingId? I don't see the need for one with what you have here.

Reply:
I think you're right.  I'll see if I can remove it.